sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -2551,6 +2552,72 @@ def test_pretty_Feedback():
     assert upretty(Feedback(tf5, tf)) == expected8
 
 
+def test_pretty_TransferFunctionMatrix():
+    tf1 = TransferFunction(x + y, x - 2*y, y)
+    tf2 = TransferFunction(x - y, x + y, y)
+    tf3 = TransferFunction(y**2 - 2*y + 1, y + 5, y)
+    tf4 = TransferFunction(y, x**2 + x + 1, y)
+    tf5 = TransferFunction(1 - x, x - y, y)
+    tf6 = TransferFunction(2, 2, y)
+    expected1 = \
+"""\
+⎡ x + y ⎤\n\
+⎢───────⎥\n\
+⎢x - 2⋅y⎥\n\
+⎢       ⎥\n\
+⎢ x - y ⎥\n\
+⎢ ───── ⎥\n\
+⎣ x + y ⎦\
+"""
+    expected2 = \
+"""\
+⎡    x + y     ⎤\n\
+⎢   ───────    ⎥\n\
+⎢   x - 2⋅y    ⎥\n\
+⎢              ⎥\n\
+⎢    x - y     ⎥\n\
+⎢    ─────     ⎥\n\
+⎢    x + y     ⎥\n\
+⎢              ⎥\n\
+⎢   2          ⎥\n\
+⎢- y  + 2⋅y - 1⎥\n\
+⎢──────────────⎥\n\
+⎣    y + 5     ⎦\
+"""
+    expected3 = \
+"""\
+⎡   x + y        x - y   ⎤\n\
+⎢  ───────       ─────   ⎥\n\
+⎢  x - 2⋅y       x + y   ⎥\n\
+⎢                        ⎥\n\
+⎢ 2                      ⎥\n\
+⎢y  - 2⋅y + 1      y     ⎥\n\
+⎢────────────  ──────────⎥\n\
+⎢   y + 5       2        ⎥\n\
+⎢              x  + x + 1⎥\n\
+⎢                        ⎥\n\
+⎢   1 - x          2     ⎥\n\
+⎢   ─────          ─     ⎥\n\
+⎣   x - y          2     ⎦\
+"""
+    expected4 = \
+"""\
+⎡    x - y        x + y       y     ⎤\n\
+⎢    ─────       ───────  ──────────⎥\n\
+⎢    x + y       x - 2⋅y   2        ⎥\n\
+⎢                         x  + x + 1⎥\n\
+⎢                                   ⎥\n\
+⎢   2                               ⎥\n\
+⎢- y  + 2⋅y - 1   x - 1      -2     ⎥\n\
+⎢──────────────   ─────      ───    ⎥\n\
+⎣    y + 5        x - y       2     ⎦\
+"""
+    assert upretty(TransferFunctionMatrix([[tf1], [tf2]])) == expected1
+    assert upretty(TransferFunctionMatrix([[tf1], [tf2], [-tf3]])) == expected2
+    assert upretty(TransferFunctionMatrix([[tf1, tf2], [tf3, tf4], [tf5, tf6]])) == expected3
+    assert upretty(TransferFunctionMatrix([[tf2, tf1, tf4], [-tf3, -tf5, -tf6]])) == expected4
```

Comment:
More tests can be added here.

Reply:
I will add more tests after reviewing process of the code is complete. That's why I have also not added any tests of other functions of `TransferFunctionMatrix` (like `subs`, `transpose` etc.). I will add them once the API changes are finalized.

Reply:
More tests are not needed. There are already too many tests. The function being tested here is a trivial forwarding method so a single test is fine.

Reply:
Maybe update `expected4` with the test in which `Series` or `Parallel` instances are passed into the nested list.